packages/jest-config/package.json

```diff
@@ -17,12 +17,12 @@
   },
   "dependencies": {
     "merge": "^2.1.1",
-    "ts-jest": "^26.5.5",
+    "ts-jest": "^27.1.2",
     "typescript": "^4.2.3"
   },
   "peerDependencies": {
     "@types/jest": "^26.0.22",
-    "jest": "^26.6.3",
+    "jest": "^27.4.5",
     "jest-coverage-thresholds-bumper": "0.0.3",
     "jest-extended": "^0.11.5",
     "jest-mock-extended": "^1.0.13"
```

Comment:
Might as well bump those to v1.1, v1.2 and v2.0 repsectively, to ensure compatibility with Jest v27.